tensorflow/core/common_runtime/gpu/gpu_cudamallocasync_allocator.h

```diff
@@ -85,10 +85,14 @@ class GpuCudaMallocAsyncAllocator : public Allocator {
 
   void SetStream(void* stream) override {
 #if TF_CUDA_MALLOC_ASYNC_SUPPORTED
-    cuda_stream_ = reinterpret_cast<CUstream>(stream);
+    cuda_stream_ = *(static_cast<CUstream*>(stream));
 #endif
   }
 
+  static int NbInstantiated() {
```

Comment:
Maybe call this `GetInstantiatedCountTestOnly`?

Reply:
Done.